src/test/java/org/jboss/tools/intellij/openshift/actions/project/CreateProjectActionTest.java

```diff
@@ -11,46 +11,41 @@
 package org.jboss.tools.intellij.openshift.actions.project;
 
 import com.intellij.openapi.actionSystem.AnAction;
-import com.intellij.openapi.actionSystem.AnActionEvent;
+import com.intellij.openapi.actionSystem.Presentation;
 import org.jboss.tools.intellij.openshift.actions.ActionTest;
-import org.jboss.tools.intellij.openshift.tree.application.ComponentNode;
-import org.jboss.tools.intellij.openshift.tree.application.NamespaceNode;
-import org.jboss.tools.intellij.openshift.tree.application.ServiceNode;
-
-import static org.mockito.Mockito.mock;
+import org.jetbrains.annotations.NotNull;
 
 public class CreateProjectActionTest extends ActionTest {
+  public CreateProjectActionTest(boolean isOpenshift) {
+    super(isOpenshift);
+  }
+
   @Override
   public AnAction getAction() {
     return new CreateProjectAction();
   }
 
   @Override
-  protected void verifyLoggedInCluster(boolean visible) {
-    assertTrue(visible);
+  protected void verifyLoggedInCluster(@NotNull Presentation presentation) {
+    assertTrue(presentation.isVisible());
+    assertEquals("New " + (isOpenshift ? "Project" : "Namespace"), presentation.getText());
   }
-
+/*
   public void testActionOnProject() {
```

Comment:
Can you please explain why you're commenting these tests?

Reply:
leftovers to be removed.